tests/test_normalize.py

```diff
@@ -445,6 +445,8 @@ def test_normalize_standardize_allsamples_compress():
 def test_normalize_spherize():
     for spherize_method in ["ZCA", "PCA", "ZCA-cor", "PCA-cor"]:
         for spherize_center in [True, False]:
+            if spherize_method in ["PCA-cor", "ZCA-cor"] and not spherize_center:
+                continue
```

Comment:
Can you add a test that enters the assertion? This is a great spot for it!

Reply:
nice